.circleci/config.yml

```diff
@@ -192,6 +190,7 @@ commands:
             EMTEST_LACKS_SOUND_HARDWARE: "1"
             EMTEST_LACKS_OFFSCREEN_CANVAS: "1" # looks broken in ff nightly
             EMTEST_DETECT_TEMPFILE_LEAKS: "0"
+            EMTEST_LACKS_THREAD_SUPPORT: "1" # wait for dom.postMessage.sharedArrayBuffer.bypassCOOP_COEP.insecure.enabled to be present on dev edition
```

Comment:
Can you put the comment on the line before?

Reply:
I did it for consistency with 2 lines earlier :) Should I change both?

Reply:
I think so yes.   Sorry for nit picking.  It might be just me but find end-of-line comments quite distracting to read and hard to format when they get to long, like this one/

Reply:
I moved both.

Reply:
Fair enough, no problem.